.vscode/tasks.json

```diff
@@ -1,10 +1,31 @@
 {
   "version": "2.0.0",
   "tasks": [
+    {
+      "label": "Setup",
+      "type": "shell",
+      "command": "${workspaceFolder}/scripts/dev-env/run.sh",
+      "windows": {
+        "command": "${workspaceFolder}/scripts/dev-env/run.cmd",
+        "args": ["./scripts/setup.sh"]
```

Comment:
does this only pass the correct args on windows?

Reply:
Bug, I fixed it.